codebuild/test_vectors/nodejs_latest.yml

```diff
@@ -0,0 +1,27 @@
+version: 0.2
+
+env:
+  variables:
+    NODE_OPTIONS: "--max-old-space-size=4096"
+    NPM_CONFIG_UNSAFE_PERM: true
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: latest
+    commands:
+      - npm ci
+      - |-
+        if [ "$PUBLISH_LOCAL" = "true" ]; then
+          npm run build
+        fi
+  build:
+    commands:
+      - npm test
+      - |-
+        if [ "$PUBLISH_LOCAL" = "true" ]; then
+          npm run verdaccio-publish
+          npm run verdaccio-verify-publish -- ci
```

Comment:
This `-- ci` part needs to be an option.
Because we want to test this with `-- ci` when test a PR.
Then when publishing we want to run with `-- ci` before we version,
and then after publish we run it again without this option to test directly from.